### PR TITLE
Revise "about" page; mathics-threejs-backend again

### PR DESCRIPTION
**Pull request by rocky**
Reinstate mathics-threejs-backend code that seems to have been lost in various merges. 

But beef up `Makefile` to copy `mathics-three-js-backend`'s `package.json`. 

From `package.json` pick out revision numbers. 

Split up "about" page into different software sections.

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -20,15 +20,16 @@ jobs:
...
+        sudo apt-get update -qq && sudo apt-get install -qq liblapack-dev llvm-dev remake nodejs npm
         python -m pip install --upgrade pip
         # Can comment out when next Mathics is released
         python -m pip install -e git://github.com/mathics/Mathics#egg=Mathics3[full]
+        npm install
         pip install -e .[full]
     - name: Install Mathics Django
       run: |
-        make develop
+        remake -x develop
```
Comment (TiagoCavalcanteTrindade) [inline .github/workflows/ubuntu.yml:31]: Won't this just let the build time slower as we need to install a new dependency?

Comment (rocky) [reply .github/workflows/ubuntu.yml:31]: How much time do you estimate it takes. 

For my part as soon as something messes up all that time is lost in my mental ability to figure out what's wrong.

Comment (TiagoCavalcanteTrindade) [reply .github/workflows/ubuntu.yml:31]: It won't take so long. I didn't know it let the packaging easier.

Comment (rocky) [reply .github/workflows/ubuntu.yml:31]: Some of this _isn't_ hypothetical. If you look at the CI history we were failing in `make develop` and I needed to figure out why. And then I realized hey, I have spent _years_ writing a tool that will help on this. Why can't I just use it (even if it takes < 5 seconds more per run).

**Pull request merged**